.github/workflows/pr-actions.yml

```diff
@@ -132,7 +133,7 @@ jobs:
             echo remote_repo=$remote_repo
             remote_branch=$(git config --get branch.${current_branch}.merge)
             echo remote_branch=$remote_branch
-            git commit -m "Results from /fix:${PR_ACTION}"
+            git commit -m "Results from /fix directive"
```

Comment:
We don't have access to `PR_ACTION` in this job, and I don't think that it's work re-parsing it, so I've simplified the message. /cc @svrnm